README.md

```diff
@@ -22,14 +22,17 @@ Prerequisite: [Node 20+](https://www.npmjs.com/package/node/v/20.11.1)
    ```shell
    npm install @coinbase/staking-client-library-ts
    ```
-2. Create and download an API key from the [Coinbase Developer Platform](https://portal.cdp.coinbase.com/access/api).
-3. Place the key named `.coinbase_cloud_api_key.json` at the root of this repository.
-4. Install necessary Typescript dependencies:
+
+2. Install necessary Typescript dependencies:
    ```shell
-    npm install -g ts-node
-    npm install -g typescript
-    ```
-5. Copy and paste one of the code samples below or any of our [provided examples](./examples/) into an `example.ts` file and run it with `ts-node` :rocket:
+   npm install -g ts-node typescript
+   ```
+
+3. Get your API keys info such as api key name and api private key from here: https://portal.cdp.coinbase.com/access/api. <br>
+   These will be used in order to set up our client later in the example code. <br>
+   For detailed instructions refer to our api key setup guide [here](https://docs.cdp.coinbase.com/developer-platform/docs/cdp-keys).
+
+4. Copy and paste one of the code samples below or any of our [provided examples](./examples/) into an `example.ts` file and run it with `ts-node` :rocket:
```

Comment:
I think we can use fewer words here 🤔.

```suggestion
3. Download an API Key from the [portal](https://portal.cdp.coinbase.com/access/api). For more information about API Keys, refer to [our docs](https://docs.cdp.coinbase.com/developer-platform/docs/cdp-keys).

4. Copy one of the code samples below into an `example.ts` file and run: :rocket:
```

Reply:
tl;dr want to move away from downloading the file completely